docs/configuration.md

```diff
@@ -327,8 +327,8 @@ _Default: Infinity_
 
 This parameter configures the maximum zoom level available. Set it to a lower value to prevent your users from zooming in too deeply.
 
-### removeOnResize
+### destroy
 
 _Default: () => {}_
 
-Function to be executed when you want to remove [`resize`](https://developer.mozilla.org/en-US/docs/Web/Events/resize) event.
+Function to be executed when you want to destroy the chart. By default, it remove `resize` event and execute the callback.
```

Comment:
s/Function to be executed when you want to destroy the chart. By default, it remove `resize` event and execute the callback./Execute this function before to removing the chart from DOM. It prevents some memory leaks due to event listeners./